README.md

```diff
@@ -136,8 +136,8 @@ curl -LsSf https://astral.sh/ruff/install.sh | sh
 powershell -c "irm https://astral.sh/ruff/install.ps1 | iex"
 
 # For a specific version.
-curl -LsSf https://astral.sh/ruff/0.6.5/install.sh | sh
-powershell -c "irm https://astral.sh/ruff/0.6.5/install.ps1 | iex"
+curl -LsSf https://astral.sh/ruff/0.6.6/install.sh | sh
+powershell -c "irm https://astral.sh/ruff/0.6.6/install.ps1 | iex"
```

Comment:
Good reminder that we should updated to the latest cargo-dist and change these to the more verbose recommendations that succeed more often.

Reply:
I can do the upgrade. I noticed that we don't use the more verbose ones in the uv README though -- is that intentional?